Cargo.toml

```diff
@@ -45,7 +45,7 @@ xxhash-c = "0.8"
 xorf = "0.7"
 angry-purple-tiger = "0"
 lorawan = { package = "lorawan", path = "lorawan" }
-semtech-udp = { version = "0", default-features=false, features=["server"] }
+semtech-udp = { version = "^0.5.2", default-features=false, features=["server"] }
```

Comment:
This should not be needed? version 0 with a cargo update would pick up the newer version

Reply:
I added a new API for getting the MAC of a downlink in `0.5.2` and it doesn't exist in `0.5.1`. I think this is better form than depending on the lock?